### DISCUSSION TITLE
Enhancement: premature birth

### DISCUSSION THREAD

**Discussion by rbeumer** [babybuddy/babybuddy#318] [Ideas]
Our son was born early (<37 weeks) so he is still in the hospital. We care for him as much as possible but there are some things that are done differently because of his age:

- feeding via a tube because he is unable to swallow until he is old enough. This should be an option for feeding type.
- Age: when baby's are premature you basically track two ages: days past since birth and how much weeks the baby would have been if it was still in the womb. My proposal would be to be able to track this separately if you mark the baby as premature.

I hope this can help other parents 🙂

**Comment by cdubz** [comment]
Congratulations! Hope all goes well with the baby.

Adding a "tube" feeding option should be fine. Some day I just need to make that field configurable...

Tracking premature babies may be a bit tough. What would you propose as different from setting the birth date as the day of birth (i.e. the early date)? If the app also tracked the anticipated date, what would it do with that information/where would it be relevant?

**Reply by fodi666** [reply]
I can imagine that if you'd add grow charts or something similar you could use that info to correct for birth date, otherwise I don't think that it makes a difference in the current way the app works

**Comment by jcgoette** [comment]
Should we add `tube`, `other`, `tube` and `other`, or just focus on making this customizable?

**Comment by TaddyJ** [comment]
Our son was born 35+3. So just over 4 weeks Premmie..
Currently using the app and have managed to convince my wife to also! :) (Honestly the hardest part of the whole thing) 

Some kind of tracking like this would definitely be handy!
I would imagine it would like like this,
Age: 6 Weeks (1 Week 4 Days Corrected)

Mainly because all of the milestones are based off their corrected age not their birth age. I.e 3 months holding head high without support.. Whereas Premmies would be (in our case) 4 + 4months.

**Comment by rbeumer** [comment]
What @TaddyJ said. We constantly calculate how old our son "really" is (the original calculated due date) to know what developmental milestones we might expect. When he gets older the difference gets less and less but when they are very young it really makes a large difference.

**Comment by cdubz** [comment]
Ok I think I’m getting the picture here. Understanding the adjusted date is not necessarily about changing or enhancing any particular data or behaviors in the app itself. It’s about helping users have a fast and simple reference.

**Reply by TaddyJ** [reply]
Pretty much.
For all of the milestones throughout a babies growth until they are 2 years of age are calculated by healthcare professionals on their corrected age.
It's just handy to have a quick reference. :)

**Comment by cdubz** [comment]
Alright I’ve created a new issue to track this in #369.
